src/lando/main/models/revision.py

```diff
@@ -73,6 +81,17 @@ def __repr__(self) -> str:
         )
         return f"<{self.__class__.__name__}: {self.id}{phab_identifier}>"
 
+    def url(self) -> str:
+        """Return a public URL for the Revision."""
+        if self.revision_id:
+            return f"{settings.PHABRICATOR_URL}/D{self.revision_id}"
+
+        if job := self.landing_jobs.first():
+            return f"{settings.SITE_URL}/landings/{job.id}#r{self.id}"
+
+        # This is not a functional URL, but this is last resort.
+        return f"{settings.SITE_URL}/#{self.id}"
```

Comment:
I don't know what this would be useful for, better to fail more obtrusively than to return a non-functional URL, or return nothing perhaps.